Collage Exercise

binkobon opened the pull request:
I used the example in the textbook to do this. In my final iteration, I
don't want the grid to repeat this much, and ideally, I want it to be of
a crop in the picture.

binkobon committed:
Collage Exercise

I used the example in the textbook to do this. In my final iteration, I
don't want the grid to repeat this much, and ideally, I want it to be of
a crop in the picture.

LadyK commented:
Hi Kimberly,

Please put a reference to the text, as well as to comment each line. This is required and will help in your understanding. It will also help your resolve one of the errors you are getting, as well as to figure out for-loops + arrays.

I hope this is still in it's infant stages. Turning in code from the book for finished work isn't good. I hope you've been developing this code since. I look forward to seeing the progress, as well as the integration of text.

Also, you are using an older version of P5. I've detailed that in several emails I've sent since the last class. And the meta tag in your index.html   ---> meta charset="UTF-8"

LadyK commented:
<meta charset="UTF-8">

binkobon committed:
Exercise 4: progress

Messed around with it more and changed it entirely.
Still wanting to add more and incorporate text

binkobon committed:
Collage: Final Version

I couldn't get some of the movements to work the way I intended, but
what I tried to go for was just a scenery with some characters I liked.

LadyK commented:
See Chapter 14 in the book for more about rotation. There are a ton of examples in there. Also, the last 2 items that I mentioned above, were not addressed. That's problematic moving forward.

binkobon committed:
Edited

Edited the first submission with comments and reference
